basil/TL/SensirionSensorBridge.py

```diff
@@ -0,0 +1,90 @@
+#
+# ------------------------------------------------------------
+# Copyright (c) All rights reserved
+# SiLab, Institute of Physics, University of Bonn
+# ------------------------------------------------------------
+#
+
+import logging
+import array
+
+from basil.TL.TransferLayer import TransferLayer
+
+logger = logging.getLogger(__name__)
+
+try:
+    from sensirion_shdlc_driver import ShdlcSerialPort, ShdlcConnection
+    from sensirion_shdlc_sensorbridge import SensorBridgePort, SensorBridgeShdlcDevice
+    from sensirion_shdlc_sensorbridge.device_errors import SensorBridgeI2cTimeoutError
+    TimeoutError = SensorBridgeI2cTimeoutError
+except ModuleNotFoundError:
+    logger.error("You have to install the package 'sensirion_shdlc_sensorbridge' to use this device.")
+    raise
+
+bridge_ports = {
+    "one": SensorBridgePort.ONE,
+    "two": SensorBridgePort.TWO,
+    "all": SensorBridgePort.ALL,
+}
+
+class SensirionSensorBridge(TransferLayer):
+    '''
+    Driver for Sensirion Sensor Bridge using the official SHDLC drivers of Sensirion.
+    The Sensirion Sensor Bridge is connected via USB and allows communication to two I2C ports.
+    '''
+
+    def __init__(self, conf):
+        super(SensirionSensorBridge, self).__init__(conf)
+
+    def init(self):
+        super(SensirionSensorBridge, self).init()
+
+        if 'port' in self._init.keys():
+            self.port = self._init['port']
+        else:
+            self.port = '/dev/ttyUSB0'
+        if 'baudrate' in self._init.keys():
+            self.baudrate = self._init['baudrate']
+        else:
+            self.baudrate = 460800
+
+        self.ser = ShdlcSerialPort(port=self.port, baudrate=self.baudrate)
+
+    def get_sensor_bridge_port(self, port):
```

Comment:
again redundant. if the dict is in the class definition, you can directly access it from the outside without a getter..